nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVReader.java

```diff
@@ -108,23 +106,14 @@
     }
 
     @OnEnabled
-    public void storeCsvFormat(final ConfigurationContext context) {
+    public void storeStaticProperties(final ConfigurationContext context) {
+        this.context = context;
+
         this.csvParser = context.getProperty(CSV_PARSER).getValue();
-        this.csvFormat = CSVUtils.createCSVFormat(context);
```

Comment:
I ran some performance tests, but I could not measure any significant difference. 
Just for sure, I changed the implementation to create a static CSV format in case no property uses EL.